vividus-plugin-web-app-playwright/src/main/java/org/vividus/ui/web/playwright/steps/ElementSteps.java

```diff
@@ -327,4 +343,63 @@ private void saveAttributeValueOfElement(Locator element, String attributeName,
                         () -> softAssert.recordFailedAssertion(
                                 String.format("The '%s' attribute does not exist", attributeName)));
     }
+
+    /**
+     * Checks that the context <b>element</b> has an expected <b>CSS properties</b>
+     * <p>The expected CSS parameters to be defined in the ExamplesTable:</p>
+     * <ul>
+     * <li><b>cssName</b> - the name of the CSS property</li>
+     * <li><b>comparisonRule</b> - String comparison rule: "is equal to", "contains", "does not contain",
+     * "matches".</li>
+     * <li><b>expectedValue</b> - expected CSS property value</li>
+     * </ul>
+     * <p>Usage example:</p>
+     * <code>
+     * <br>Then context element has CSS properties:
+     * <br>|cssName |comparisonRule |expectedValue |
+     * <br>|border  |contains       |solid         |
+     * </code>
+     *
+     * @param parameters The parameters of the expected CSS properties to set as ExamplesTable
+     */
+    @Then("context element has CSS properties:$parameters")
```

Comment:
Also it makes sense to move this step `ElementCssSteps` class (to keep consistency with Selenium plugin)

Reply:
done